PumpBot.py

```diff
@@ -45,6 +50,25 @@ def log(information):
 manualBTC = float(data['manualBTC'])
 profitMargin = float(data['profitMargin']) / 100
 stopLoss = float(data['stopLoss'])
+currentVersion = float(data['currentVersion'])
+
+# check we have the latest version
+url = 'https://raw.githubusercontent.com/fj317/PumpBot/master/config.json'
+urllib.request.urlretrieve(url, 'version.json')
+try:
```

Comment:
This try-catch will always fair since version.json does not exist?

Reply:
It will only fail if the latest config.json file can not be downloaded and renamed to version.json

Reply:
The urlretrieve renames the downloaded file into version.json